content/applications/inventory_and_mrp/inventory/management/warehouses/use_locations.rst

```diff
@@ -0,0 +1,58 @@
+=================================
+Use locations to manage inventory
+=================================
+
+A *location* is a specific space within a warehouse. This can be a shelf, room, aisle, etc. There
+are three types of locations in Odoo:
+
+- *Physical locations* are spaces within a warehouse owned by the user's company. These can be a
+  an area where items are stored like an aisle or shelf, or an area where operations take place, like
+  loading and unloading bays.
+- *Partner locations* are the same as physical locations except that they exist within the warehouse
+  of a customer or vendor.
+- *Virtual locations* are locations that do not exist physically, but where items that are not in
+  inventory can be placed. These can be items that have not yet entered inventory, like products that
+  are on the way to a warehouse, or items that are no longer in inventory due to loss or other
+  factors.
+
+.. important::
+  In order to use locations, the :guilabel:`Storage Locations` setting must be enabled from the
+  :guilabel:`Inventory` module by selecting :menuselection:`Configuration --> Settings` and clicking
+  the checkbox under the Warehouse heading.
```

Comment:
the text within this admonition block needs to be indented by one more space. It should align with the `i` in *important*.